mixer/pkg/il/opcode.go

```diff
@@ -770,6 +866,137 @@ var opCodeInfos = map[Opcode]opcodeInfo{
 		// The name of the attribute.
 		OpcodeArgString,
 	}},
+
+	// EqS pops two string values from the stack and compare for equality. If the strings are
+	// equal then it pushes 1 into the stack, otherwise it pushes 0.
+	LtS: {name: "LtS", keyword: "lt_s"},
+
+	// EqS pops two string values from the stack and compare for equality. If the strings are
+	// equal then it pushes 1 into the stack, otherwise it pushes 0.
```

Comment:
These are all copy-pasted comments. Perhaps, rewrite them or delete them.

Reply:
Thanks for the pointer!
I sorted and fixed the comments in the opcode constants above, but forgot these.